modules/storage-page-memory/src/main/java/org/apache/ignite/internal/storage/pagememory/PersistentPageMemoryDataRegion.java

```diff
@@ -437,13 +440,13 @@ private void checkDataRegionStarted() {
     /** Adds a table storage to the data region. */
     @VisibleForTesting
     public void addTableStorage(PersistentPageMemoryTableStorage tableStorage) {
-        boolean add = tableStorages.add(tableStorage);
+        PersistentPageMemoryTableStorage old = tableStorages.put(tableStorage.getTableId(), tableStorage);
 
-        assert add : tableStorage.getTableId();
+        assert old == null : tableStorage.getTableId();
```

Comment:
Even if this method is only used in tests, it would still be nice to add a meaningful message.